magefile.go

```diff
@@ -69,15 +70,41 @@ func Lint() error {
 		return err
 	}
 
-	if err := sh.RunV("go", "mod", "tidy"); err != nil {
-		return err
-	}
+	if err := filepath.WalkDir(".", func(path string, d os.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
 
-	if err := sh.RunV("go", "work", "sync"); err != nil {
+		if !d.IsDir() {
+			return nil
+		}
+
+		if _, err := os.Stat(filepath.Join(path, "go.work")); err == nil {
```

Comment:
I think this can be outside the walkdir itself

Reply:
Makes sense.